Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -376,23 +376,33 @@ public class PBXProjGenerator {
 
     func sortGroups(group: PBXGroup) {
         // sort children
-        let children = group.children
-            .sorted { child1, child2 in
-                let sortOrder1 = child1.getSortOrder(groupSortPosition: project.options.groupSortPosition)
-                let sortOrder2 = child2.getSortOrder(groupSortPosition: project.options.groupSortPosition)
-
-                if sortOrder1 != sortOrder2 {
-                    return sortOrder1 < sortOrder2
-                } else {
-                    if child1.nameOrPath != child2.nameOrPath {
-                        return child1.nameOrPath.localizedStandardCompare(child2.nameOrPath) == .orderedAscending
+        if let groupOrder = project.options.groupsOrder.first(where: { group.nameOrPath == $0.pattern || group.nameOrPath.isMatch(to: $0.pattern) }) {
```

Comment:
`nameOrPath` evaluates to `name ?? path ?? ""`. That means groups who have a different path than just the name won't be supported.
In either case doing something like `Group1/Group2/*` won't work as the name is just the end and the path is a relative path.

Reply:
Okay, so should I to use just `name`?

Reply:
@yonaskolb I need your help. I don't understand what is the correct way to do it.